src/main/java/se/sundsvall/emailreader/integration/ews/EWSIntegration.java

```diff
@@ -130,7 +128,10 @@ private Folder findFolder(final String emailAddress, final String folderName) th
 		final var findFoldersResults = exchangeService.findFolders(folderId, searchFilter, folderView);
 
 		if ((findFoldersResults == null) || (findFoldersResults.getFolders().size() != 1)) {
```

Comment:
This will probably work "as is", but it would feel better to have this logic:

```
if ((findFoldersResults != null) && (findFoldersResults.getFolders().size() > 1)) {
   throw new IllegalArgumentException("Could not determine a unique folder with the name: " + folderName);;
}

if ((findFoldersResults == null) || (findFoldersResults.getFolders().size() < 1)) {
  // "Your create folder"-logic here...
}
```

I.e. it feels weird to create a new folder if we for some reason get 2 search hits.